index.html

```diff
@@ -0,0 +1,447 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="stylesheet" href="./css/main.css" />
+    <link rel="icon" href="./favicon.ico" />
+    <title>Aziz Mejri - Portfolio</title>
+  </head>
+  <body>
+    <nav>
+      <div id="nav-bar">
+        <div class="nav-cont container header">
+          <a id="logo" href="#">Aziz Mejri</a>
+          <button type="submit" class="menu-bars">
```

Comment:
Its a good practice to only place the links inside the `nav` element. Kindly place the logo outside the `nav` element but still inside the `header`. This helps in accessibility and styling the links 💯